spec/models/course_spec.rb

```diff
@@ -281,6 +284,30 @@
       end
     end
 
+    describe '#not_new' do
+      let(:new_course) do
+        create(:course, site_statuses: [new_site_status])
+      end
+
+      let(:findable_course_with_new_site) do
+        create(:course, site_statuses: [suspended, new_site_status2])
```

Comment:
A course must have a site_status that's running to be findable.

Reply:
Resolved in https://github.com/DFE-Digital/manage-courses-backend/pull/645